src/components/global-toolbar/global-toolbar.js

```diff
@@ -58,6 +61,7 @@ export const GlobalToolbar = ({
               className={
                 'pipeline-menu-button--large pipeline-menu-button--link'
               }
+              dataTest={'View your experiments'}
```

Comment:
Does Cypress suggest targeting elements with the `aria-label` attribute? This could be a place to do that, if so.

Reply:
I have used that initially but it is not recommended as there can be duplicates using aria